lte/gateway/c/oai/common/common_types.h

```diff
@@ -228,7 +235,13 @@ typedef union {
   uint8_t imeisv[IMEISV_DIGITS_MAX];
 } me_identity_t;
 
+typedef enum {
+  BPS  = 0,
+  KBPS = 1,
+} apn_ambr_bits_unit_t;
+
 typedef struct {
+  apn_ambr_bits_unit_t bitsunit;
```

Comment:
make field name bitsunit consistent with the proto definition i.e., `bitrate_units`.

Reply:
Resolved